app/src/main/cpp/src/3rd_party/android-helpers/openssl/build_openssl.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash \
+
+ANDROID_NDK=$1
+API_LEVEL=23
```

Comment:
@VadymLuchko can we get current API level from cmake?

Reply:
done in 048ae59c19ca4e61f504361cf27bd9190b2e5830